### PR TITLE
Add LayerGroup permissions and fix permissions management of layers

### PR DESCRIPTION
**Pull request by nahuelange**
Warning: this renamed the permission can_update_features_properties to can_manage_layers

### CODE REVIEW

File: geostore/permissions.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        else:
+            return request.user.has_perm('geostore.can_manage_layers')
+
+    def has_object_permission(self, request, view, obj):
+        if request.user.is_superuser:
+            return True
+
+        has_perm = request.user.has_perm('geostore.can_manage_layers')
+
+        if obj.authorized_groups.count():
```
Comment (submarcos) [inline geostore/permissions.py]: don't use count if you plan to iterate on qs

https://docs.djangoproject.com/en/2.2/topics/db/optimization/#don-t-overuse-count-and-exists

Comment (nahuelange) [reply geostore/permissions.py]: In fact we use it only if it contains something, exists() is a better way.

File: geostore/models.py [hunk 2]
```diff
@@ -77,6 +78,8 @@ class Layer(BaseUpdatableModel):
     name = models.CharField(max_length=256, unique=True, default=uuid.uuid4)
     schema = JSONField(default=dict, blank=True, validators=[validate_json_schema])
     geom_type = models.IntegerField(choices=GeometryTypes.choices(), null=True)
+    authorized_groups = models.ManyToManyField(Group, related_name='authorized_layers')
```
Comment (submarcos) [inline geostore/models.py]: blank=True (authorized_groups will not be required in admin section)

Comment (nahuelange) [reply geostore/models.py]: fixed

**Pull request merged**